specification/maps/data-plane/Microsoft.Maps/Alias/preview/1.0/alias.json

```diff
@@ -5,7 +5,7 @@
     "version": "1.0",
     "description": "APIs for managing aliases in Azure Maps."
   },
-  "host": "atlas.microsoft.com",
```

Comment:
there is a way to add {geo} parameterized host name
1) replace this "host" line with
"x-ms-parameterized-host": {
    "hostTemplate": "{geography}.atlas.microsoft.com",
    "parameters": [
      {
        "$ref": "#/parameters/GeographicResourceLocation"
      }
    ]
  },

2) add this in the "parameters" section
    "GeographicResourceLocation": {
      "name": "geography",
      "description": "This parameter specifies where the Azure Maps Creator resource is located.  Valid values are us and eu.",
      "in": "path",
      "required": true,
      "type": "string",
      "default": "us",
      "enum": [
        "us",
        "eu"
      ],
      "x-ms-enum": {
        "name": "GeographicResourceLocation",
        "modelAsString": true,
        "values": [
          {
            "value": "us",
            "description": "Used to access an Azure Maps Creator resource in the United States"
          },
          {
            "value": "eu",
            "description": "Used to access an Azure Maps Creator resource in Europe"
          }
        ]
      },
      "x-ms-parameter-location": "client"
    },